.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
This kind of thing belongs in a user specific .gitignore file http://stackoverflow.com/questions/5724455/can-i-make-a-user-specific-gitignore-file